Verbatim `util/seqset.v` section of patches/prosa-v06-rocq93-official-proof-closure-common.patch
(sha256 6819bfa9ac0bb9cfd6bfc7182d57e22463cf7a57b38ce9af89c8e7de96868050), split out for classic official closures.
--- a/util/seqset.v
+++ b/util/seqset.v
@@ -21,7 +21,7 @@
   HB.instance Definition _  setSubType := [isSub for _set_seq].
   HB.instance Definition _ := [Equality of set by <:].
   Canonical Structure mem_set_predType := PredType (fun (l : set) => mem_seq (_set_seq l)).
-  Definition set_of of phant T := set.
+  Definition set_of (_ : phant T) := set.
 
 End SeqSet.
 Set Warnings "redundant-canonical-projection".
